Makefile

```diff
@@ -1,11 +1,22 @@
+export GO15VENDOREXPERIMENT:=1
```

Comment:
You might find parts of https://github.com/coreos/coreos-baremetal/blob/master/test useful.